includes/discovery/sensors/current/junos.inc.php

```diff
@@ -16,7 +16,7 @@
     $multiplier = 1;
     $divisor    = 1000000;
     foreach ($junos_oids as $index => $entry) {
-        if (is_numeric($entry['jnxDomCurrentTxLaserBiasCurrent'])) {
+        if (is_numeric($entry['jnxDomCurrentTxLaserBiasCurrent']) && $entry['jnxDomCurrentTxLaserBiasCurrent'] && $entry['jnxDomCurrentTxLaserBiasCurrentLowAlarmThreshold'] != 0) {
```

Comment:
Yes but why not only skip jnxDomCurrentTxLaserBiasCurrent? We can calculate the threshold so just null it if it's invalid and allow the sensor to be discovered.